src/internals/numbers/impl/digits/division.ts

```diff
@@ -59,20 +59,27 @@ export type DivModDigits<
   Remainder: infer R extends Digit[];
 }
   ? A extends [infer A1 extends Digit, ...infer AR extends Digit[]]
-    ? DivModDigits<AR, TrimZeros<[...R, A1]>, M, [...Q, B]>
+    ? _DivModDigits<AR, TrimZeros<[...R, A1]>, M, [...Q, B]>
     : { Quotient: [...Q, B]; Remainder: R }
   : never;
 
 export type DivDigits<N extends Digit[], M extends Digit[]> = TruncateWith<
   N,
   M
 > extends [infer A extends Digit[], infer D extends Digit[]]
-  ? DivModDigits<A, D, M>["Quotient"]
+  ? _DivModDigits<A, D, M>["Quotient"]
   : never;
 
 export type ModDigits<N extends Digit[], M extends Digit[]> = TruncateWith<
   N,
   M
 > extends [infer A extends Digit[], infer D extends Digit[]]
-  ? DivModDigits<A, D, M>["Remainder"]
+  ? _DivModDigits<A, D, M>["Remainder"]
+  : never;
+
+export type DivModDigits<N extends Digit[], M extends Digit[]> = TruncateWith<
```

Comment:
Hello, not including the `M` type param here seems to have broken `src/internals/numbers/impl/digits/power.ts` on line 14.

Reply:
thanks, fixing it now